iothub_client/src/iothubtransport_amqp_device.c

```diff
@@ -1668,3 +1692,47 @@ int device_unsubscribe_for_twin_updates(AMQP_DEVICE_HANDLE handle)
 
     return result;
 }
+
+int device_get_twin_async(AMQP_DEVICE_HANDLE handle, DEVICE_TWIN_UPDATE_RECEIVED_CALLBACK on_device_get_twin_completed_callback, void* context)
+{
+    int result;
+
+    // Codes_SRS_DEVICE_09_152: [If `handle` or `on_device_get_twin_completed_callback` are NULL, device_get_twin_async shall return a non-zero result]
+    if (handle == NULL || on_device_get_twin_completed_callback == NULL)
```

Comment:
Do we need check for context?

Reply:
We do not check for context, that's an user choice.